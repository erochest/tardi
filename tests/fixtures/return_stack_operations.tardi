42 >r
r@
r>
+

1
2
3
>r
swap
r>
swap