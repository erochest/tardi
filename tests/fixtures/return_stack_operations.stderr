84
2
3
1
